src/containers/CallIn/CallIn.js

```diff
@@ -393,4 +399,4 @@ class CallIn extends Component {
     }
 }
 
-export default CallIn;
+export default connect(null, {logCall})(CallIn);
```

Comment:
I'm trying to figure this out, but do we actually need to connect if we're not actually mapping props?